sktime/datatypes/_adapter/tests/test_gluonts.py

```diff
@@ -55,6 +55,29 @@ def test_pandas_to_ListDataset(pandas_df):
         np.testing.assert_allclose(group_data.values, generated_list[idx]["target"])
         idx += 1
 
+    try:
```

Comment:
great! you do not need to do try/except, this is already taking care of by the marker at the top of the test.